### DISCUSSION TITLE
Got the build done!

### DISCUSSION THREAD

**Discussion by SoloBSD** [bruddog/Tecmo_Super_Bowl_NES_Disassembly#1] [General]
Hi, 
I was able to make the build successfully. So, what's next?
I obtained a .bin file, how do I run it?
What does it contain? the game?

**Comment by bruddog** [comment]
Yes,

The .bin is the complete game. You need to rename the .bin extension to .nes You will need a standard NES emulator to run it.

**Reply by SoloBSD** [reply]
Got it.
Thank you very much!
